README.md

```diff
@@ -1,64 +1,228 @@
-# GYFT - Get Your Freaking Timetable
+<div id="top"></div>
 
-Gets your timetable from ERP and adds it to your **Google Calendar** or gives you an **ICS file** which you can add in any common calendar application.
+<!-- PROJECT SHIELDS -->
+<!-- https://www.markdownguide.org/basic-syntax/#reference-style-links-->
+<div align="center">
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
+[![LinkedIn][linkedin-shield]][linkedin-url]
```

Comment:
This is redirecting to Arpit's LinkedIn page. This could be removed, or just replaced with MetaKGP's LinkedIn

Reply:
MetaKGP doesn't have a LinkedIn at the moment. So we can link to the wiki maybe.